tealium/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 group 'com.tealium'
-version '2.6.2'
+version '2.6.3'
```

Comment:
There are also version numbers in `tealium.dart` and the `podspec` file that need updating.